src/Kiota.Builder/CodeDOM/CodeNamespace.cs

```diff
@@ -44,6 +43,11 @@ public CodeFile TryAddCodeFile(string fileName, params CodeElement[] children)
         return file;
     }
 
+    public CodeFile TryAddCodeFile(string fileName, params CodeElement[] children)
+    {
+        return TryAddCodeFileWithChildren(fileName, children);
+    }
+
```

Comment:
```suggestion
```
I don't think we need the renamed overload here. The children can still be passed directly as an array to the original function. 
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/params